client/components/SocialClub/ClubInfo.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
+import { Link } from 'react-router';
 
-function ClubInfo({ joinClub }) {
+function ClubInfo({ joinClub, club }) {
   return (
     <div className="click-bait">
       <div className="container">
         <div className="mini-header">
           <div className="title">
-            Swimming Meetups
-            <small>For those who no matter what, stay afloat...</small>
+            <h4 className="club-link">
+              <Link to={`/clubs/${club.id}`}>
+                {club.name}
+              </Link>
+            </h4>
+            <small>{club.description}</small>
           </div>
           <a
```

Comment:
Can we remove these redundant anchor tags? You can check [this resource](https://css-tricks.com/when-and-when-not-to-use-an-anchor-tag/) to know when to use it. Just noticed we have been using it the wrong way lol.

Reply:
Ok... noted